### PR TITLE
Restiction of duplicate votes

### PR DESCRIPTION
**Pull request by Maliek**
Druk op knop.

### CODE REVIEW

File: src/main/resources/templates/error/400.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "../layouts/main.twig" %}
+
+{% block pageTitle %}Already voted - Pollistics{% endblock %}
+{% block description %}Pollistics encountered an error: 400 Already Voted{% endblock %}
```
Comment (Haroenv) [inline src/main/resources/templates/error/400.twig]: Technically, 400 means Bad Request. This is a good request, but it's not allowed, so I'd go with [403](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#403)

>The server understood the request, but is refusing to fulfill it. Authorization will not help and the request SHOULD NOT be repeated. If the request method was not HEAD and the server wishes to make public why the request has not been fulfilled, it SHOULD describe the reason for the refusal in the entity. If the server does not wish to make this information available to the client, the status code 404 (Not Found) can be used instead.

>### Wikipedia
>The request was a legal request, but the server is refusing to respond to it. Unlike a 401 Unauthorized response, authenticating will make no difference.

> Error code for user not authorized to perform the operation or the resource is unavailable for some reason (e.g. time constraints, etc.).

File: src/main/java/com/pollistics/controllers/PollController.java [hunk 2]
```diff
@@ -45,10 +55,30 @@ public String createPoll(HttpServletRequest request, Model model) {
 	}
 
 	@PostMapping(value = "/polls/vote/{pollId}")
-	public String voteOptions(@PathVariable String pollId, HttpServletRequest request, Model model) {
-		String option = request.getParameter("option");
-		Poll p = pollService.getPoll(pollId);
-		pollService.voteOption(p, option);
-		return "redirect:/" + pollId;
+	public String voteOptions(@CookieValue(value = "id", defaultValue = "") String cookieIdValue, @PathVariable String pollId, HttpServletRequest request, HttpServletResponse response, Model model) {
+		if (cookie.getValue().contains(pollId)) {
```
Comment (elsmr) [inline src/main/java/com/pollistics/controllers/PollController.java:59]: Maybe we could seperate all this cookie logic into a seperate class or method later

**Review by elsmr — APPROVED** [review]
🔥 🔥 🔥

File: src/main/java/com/pollistics/controllers/PollController.java (hunk 2 above)
Comment (Haroenv) [reply src/main/java/com/pollistics/controllers/PollController.java:59]: MAYBE MAYBE

**Comment by Haroenv** [comment]
FYI, coverage is now at 93%

**Comment by elsmr** [comment]
😠

**Pull request merged**